Time/beats.1s.lisp

```diff
@@ -0,0 +1,16 @@
+#!/usr/local/bin/clisp
+
+; <bitbar.title>Beats Time</bitbar.title>
+; <bitbar.version>v1.0</bitbar.version>
+; <bitbar.author>Jannis Segebrecht</bitbar.author>
+; <bitbar.author.github>queitsch</bitbar.author.github>
+; <bitbar.desc>Displays Swatch .beats time.</bitbar.desc>
+; <bitbar.dependencies>clisp</bitbar.dependencies>
```

Comment:
It looks like an image is missing here. It would be great if you could please add a screenshot of this plugin.